### ISSUE TITLE
The team photos need some sort of revision.

### ISSUE THREAD

**Issue by devinivy** [BigRoomStudios/bigroomstudios.com#9]
Justin and I have talked about this.  Here are the particular issues,
1. Brian O's photo is way too dark.
2. Sam F., Tim, and Mike's photo are way to green/yellow.
3. Some of the photos match in style too closely (related to 2).  Ideally we adhere to neutral/mishmash styles of team photos or uniform style team photos.  I have heard the suggestion of converting them all to b&w.
4. @tmateosian possibly wants to nix some photos for their content.

**Comment by cbracco** [comment]
Agree 100% ... this was a rushed because SXSW. Let's make 'em purdy!

**Comment by thebeline** [comment]
I would agree with Devin in that they should all be distinctly stylistically different, or stylistically uniform.  If different, then yes, converting to Black and White would be an acceptable way of making them all fit together.

If we care going to go with stylistically similar, then ideally we would want to re-shoot all the photos, using the same camera and similar lighting conditions, and treat them all with the same effects.

Personally, I am partial to keeping them stylistically similar, as if we go with the "mishmash" it introduces too many variables which will just complicate photo selection (such as we are experiencing now).

**Comment by devinivy** [comment]
For me the issue is that a group of photos fit together, but the rest do not fit with them.  A mishmash would be fine with me, especially given available dev/designer time.

**Comment by cbracco** [comment]
We wanted a mishmash, but the problem was that we received a very wide range of quality across photos (too small, grainy, over/underexposed, heavily filtered, etc). Most of us simply do not have decent photographs of ourselves, so it might be worth considering taking everyone's photos in that blurred background style that you see on mike, brian o, liam, etc. Either that, or we go fishing for photos from everyone again.

**Comment by cbracco** [comment]
Could be neat to have some kind of general theme, like this, too: http://fuzzco.com/office/who/
